portal_objects/metaworkflows/SV_germline_hg19_vcf.json

```diff
@@ -0,0 +1,590 @@
+  "version": "VERSION",
+  "accession": "GAPMWF15HC8A",
+  "description": "Downstream pipeline for SV for trio, family or proband only (hg19 based VCF)",
+  "uuid": "c492dc85-c2a3-45a3-86e6-def236b120d8",
+  "project": "PROJECT_UUID",
+  "institution": "INSTITUTION_UUID",
+  "previous_version": [
+    "f0b87c91-a3f8-487c-a513-64a61cf85ad1"
+  ],
+  "input": [
+    {
+      "argument_name": "input_vcfs",
+      "argument_type": "file",
+      "dimensionality": 1
+    },
+    {
+      "argument_name": "sample_names",
+      "argument_type": "parameter",
+      "value_type": "json"
+    },
+    {
+      "argument_name": "ref_fasta",
+      "argument_type": "file",
+      "files": [
+        {
+          "file": "1936f246-22e1-45dc-bb5c-9cfd55537fe7"
+        }
+      ]
+    },
+    {
+      "argument_name": "callRegions",
+      "argument_type": "file",
+      "files": [
+        {
+          "file": "6512e951-8fd6-4e58-8de4-135e814de175"
+        }
+      ]
+    },
+    {
+      "argument_name": "cytoband",
+      "argument_type": "file",
+      "files": [
+        {
+          "file": "3986849e-c96a-4add-82f8-eab605f193fd"
+        }
+      ]
+    },
+    {
+      "argument_name": "chainfile_hg38_hg19",
+      "argument_type": "file",
+      "files": [
+        {
+          "file": "297c872a-5b6b-4fc3-83d3-f4a853f8805c"
+        }
+      ]
+    },
+    {
+      "argument_name": "vep",
+      "argument_type": "file",
+      "files": [
+        {
+          "file": "ea103486-b65a-4439-9d0b-1186f8e59388"
+        }
+      ]
+    },
+    {
+      "argument_name": "gnomAD_SV",
+      "argument_type": "file",
+      "files": [
+        {
+          "file": "07ed6afe-b9e4-44c5-a9ca-99b99c93cbdf"
+        }
+      ]
+    },
+    {
+      "argument_name": "genes",
+      "argument_type": "file",
+      "files": [
+        {
+          "file": "84f2bb24-edd7-459b-ab89-0a21866d7826"
+        }
+      ]
+    },
+    {
+      "argument_name": "dirPath20vcf",
+      "argument_type": "file",
+      "files": [
+        {
+          "file": "cd647c0c-ac11-46db-9c51-bfe238e9ac13"
+        }
+      ]
+    },
+    {
+      "argument_name": "recip",
+      "argument_type": "parameter",
+      "value": "0.8",
+      "value_type": "float"
+    },
+    {
+      "argument_name": "max_unrelated",
+      "argument_type": "parameter",
+      "value": "1",
+      "value_type": "integer"
+    },
+    {
+      "argument_name": "wiggle",
+      "argument_type": "parameter",
+      "value": "50",
+      "value_type": "integer"
+    },
+    {
+      "argument_name": "max_length",
+      "argument_type": "parameter",
+      "value": "10000000",
+      "value_type": "integer"
+    },
+    {
+      "argument_name": "SV_types",
+      "argument_type": "parameter",
+      "value": "[\"DEL\", \"DUP\"]",
+      "value_type": "json"
+    },
+    {
+      "argument_name": "chainfile_hg19_hg38",
+      "argument_type": "file",
+      "files": [
+        {
```

Comment:
can we move this before the parameters? so basically to keep things organized we want to have the placeholder arguments, the reference files, the parameters. Probably same in the other two metaworfklow?